apps/nextjs/src/lib/hooks/use-sidebar.tsx

```diff
@@ -27,7 +27,7 @@ interface SidebarProviderProps {
 }
 
 export function SidebarProvider({ children }: SidebarProviderProps) {
-  const [isSidebarOpen, setSidebarOpen] = React.useState(true);
+  const [isSidebarOpen, setSidebarOpen] = React.useState(false);
```

Comment:
Default was open? 🤔

Reply:
The default was "the page doesn't render until we've looked at localstorage", which I suppose was falsey

I don't really know why the sidebar state would even need to be in localstorage though?

Reply:
🤷